app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,51 @@
+class ReviewsController < ApplicationController
+  # def index
+  #   @reviews = Review.all
+  # end
+  #
+  # def show
+  #   @book = Book.find(params[:id])
+  #   @reviews = @book.reviews
+  # end
+
+  RATINGS = [1, 2, 3, 4, 5]
+
+  def new
+    @book = Book.find(params[:book_id])
+    @review = Review.new
+    @ratings_collection = RATINGS
```

Comment:
Nice use of the constant! There may be a better/ActiveRecord Helper way of doing this, but you can always refactor later